underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -38,35 +74,353 @@
+    LOG.debug("close");
+  }
+
+  @Override
+  public void connectFromMaster(TachyonConf conf, String hostname) {
+    LOG.debug("connect from master");
+  }
+
+  @Override
+  public void connectFromWorker(TachyonConf conf, String hostname) {
+    LOG.debug("connect from worker");
+  }
+
+  @Override
+  public OutputStream create(String path) throws IOException {
+    LOG.debug("Create method: {}", path);
+    String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    SwiftOutputStream out = SwiftDirectClient.PUT(mAccess, newPath);
+    return out;
+  }
+
+  @Override
+  public OutputStream create(String path,
+      int blockSizeByte) throws IOException {
+    LOG.warn("Create with block size is not supported"
+        + "with SwiftDirectUnderFileSystem. Block size will be ignored.");
+    return create(path);
+  }
+
+  @Override
+  public OutputStream create(String path, short replication, int blockSizeByte) throws IOException {
+    LOG.warn("Create with block size and replication is not"
+        + "supported with SwiftDirectUnderFileSystem."
+        + " Block size and replication will be ignored.");
+    return create(path);
+  }
+
+  /* @inheritDoc
+   * @see tachyon.underfs.UnderFileSystem#delete(java.lang.String, boolean).
+   * recursive will delete all objects with given prefix.
+   * parent will not be deleted.
+   * Method always returns {@code true}.
+   */
+  @Override
+  public boolean delete(String path, boolean recursive) throws IOException {
+    LOG.debug("Delete method: {}, recursive {}", path, recursive);
+    String strippedPath = stripPrefixIfPresent(path);
+    Container c = mAccount.getContainer(mContainerName);
+    if (recursive) {
+      strippedPath = makeQualifiedPath(strippedPath);
+      PaginationMap paginationMap = c.getPaginationMap(strippedPath, 100);
+      for (int page = 0; page < paginationMap.getNumberOfPages(); page++) {
+        for (StoredObject obj : c.list(paginationMap, page)) {
+          if (obj.exists()) {
+            obj.delete();
+          }
+        }
+      }
+    }
+    StoredObject so = c.getObject(strippedPath);
+    if (so.exists()) {
+      so.delete();
+    }
+    return true;
+  }
+
+  @Override
+  public boolean exists(String path) throws IOException {
+    String newPath = stripPrefixIfPresent(path);
+    return isObjectExists(newPath) ;
   }
 
   /**
-   * Prepares the Hadoop configuration necessary to successfully obtain a {@link FileSystem}
-   * instance that can access the provided path
-   * <p>
-   * Derived implementations that work with specialised Hadoop {@linkplain FileSystem} API
-   * compatible implementations can override this method to add implementation specific
-   * configuration necessary for obtaining a usable {@linkplain FileSystem} instance.
-   * </p>
-   *
-   * @param path File system path
-   * @param config Hadoop Configuration
+   * Checks if the object exists.
+   * @param path the key to get the object details of
```

Comment:
there should be a blank line between method description and parameter descriptions